include/edata.h

```diff
@@ -19,14 +19,15 @@ class ExpData {
 
     ~ExpData();
 
-    /** observed data */
+    /** observed data (dimension: nytrue x nt, column-major) */
```

Comment:
are you sure? in matlab (column major) this is supposed to be nt x nytrue.

Reply:
You are absolutely right.